### PR TITLE
Ashley's Style

### PR DESCRIPTION
**Pull request by ashleymmedina**

### CODE REVIEW

File: index.html
Comment (MattheuJ) [inline index.html:1]: looks good, but not as good as mine.

lol.

**Review by MattheuJ — APPROVED** [review]
Good

**Review by julz08 — APPROVED** [review]
10/10 would approve 💯

File: index.html
Comment (julz08) [reply index.html:1]: Be so serious

**Review by FRLOCS — APPROVED** [review]
beutiful

File: index.html
Comment (ashleymmedina) [reply index.html:1]: Bro woke up sassy today. It's good though, keep dreaming and imagining things :))

**Comment by ashleymmedina** [comment]
> 10/10 would approve 💯

Thank you !!! I'm glad you liked it, unlike other people 🙄

**Pull request merged**